ciao-launcher/main.go

```diff
@@ -528,6 +542,14 @@ func startLauncher() int {
 	timeoutCh := make(chan struct{})
 	signal.Notify(signalCh, syscall.SIGINT, syscall.SIGTERM, syscall.SIGHUP)
 
+	configCh := make(chan payloads.Configure)
+	go connectToServer(doneCh, statusCh, configCh)
+	clusterConfig := <-configCh
```

Comment:
@obedmr This is a good point.  There'll be a race condition if we don't also delay the start of the overseer until after Dial has completed and we've retrieved our ClusterConfiguration and initialised the 4 global variables.  I think this should work, although I haven't tried.